tests/gui/builder/control/test_html.py

```diff
@@ -20,6 +20,8 @@ def test_html_builder(gui: Gui, test_client, helpers):
         with tgb.html("p", "This is a paragraph.", style="color:green;"):
             tgb.html("a", "a text", href="https://www.w3schools.com", target="_blank")
             tgb.html("br")
+            tgb.html("BR")
```

Comment:
?

Reply:
<BR> is valid html but to differentiate html tags from Taipy, a `lower()` is applied to all html tags